atomic_reactor/schemas/config.json

```diff
@@ -734,6 +734,18 @@
         "additionalProperties": false,
         "required": ["name", "value"]
       }
+    },
+    "image_size_limitation": {
```

Comment:
I'd prefer `image_size_limit` as the name for this.

To me, "limitation" sounds like something that is out of your control (technological limitation etc.).